test/integration/scheduler/scheduler_test.go

```diff
@@ -40,10 +40,6 @@ var _ = ginkgo.Describe("Scheduler", func() {
 
 	var (
 		ns                  *corev1.Namespace
-		prodClusterQ        *kueue.ClusterQueue
```

Comment:
Didn't change any test logic here but do some refactoring works so that we'll delete namespaces first then we can delete the clusterQueue successfully. Besides, divide original tests into four parts of `ginkgo.Context` which helps to organize the tests better.

Reply:
hold for #290